### PR TITLE
add(components): Tailwind CSS Components

### PR DESCRIPTION
**Pull request by besoeasy**
Added : https://github.com/besoeasy/tailwind-components

| Name                 | Link                 |
| -------------------- | -------------------- |
|Tailwind CSS Components|https://github.com/besoeasy/tailwind-components|

Open Source Tailwind CSS component collection.

---

- [x] My item is in the right category
- [x] My item is logically grouped below similar items
- [x] My item's name and description respects the conventions of the list
- [x] My item is awesome
- [x] My item is in line with the [Tailwind brand usage guidelines](https://tailwindcss.com/brand#usage)
- [x] I have read and followed the [contribution guidelines](https://github.com/aniftyco/awesome-tailwindcss/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: README.md
Comment (joshmanders) [inline README.md:277]:
```suggestion
- 🧩 [Tailwind CSS Components](https://github.com/besoeasy/tailwind-components) - Collection of open source components for Tailwind CSS.
```

**Review by joshmanders — CHANGES_REQUESTED** [review]
I'm gonna hold off on merging this until you flesh out the components a bit more. It's kind of lacking and really doesn't hit the "Is this truly AWESOME" requirement, yet!

So please continue fleshing it out, adding some docs/better page to view them, and ping me on this PR when ready and we will merge in.

**Comment by besoeasy** [comment]
@joshmanders the thing is this is not like other component library 

The main purpose is to have components references for vueJS & ReactJS Developers.


1. Users submit the component
2. All components and all versions of them are stored on IPFS (so even if users submit new component for newer tailwind, we still have old one)
3. The library is named 1.html 2.html due to easy prototyping using linters. 
(example - loading all team components, automatically and in single page and pick the one you like)
4.

So  **"Is this truly AWESOME"** is relative to the users
1. Its awesome for someone building a static website
2. Maybe Not awesome for someone building a APP or Admin Panel

Please consider !

**Comment by joshmanders** [comment]
@besoeasy unfortunately we have to be more strict on what is considered awesome otherwise this will become a dumping ground for EVERYTHING Tailwind related.

Right now it's very cumbersome to view stuff and you really don't know what it offers until you click through at least 3 different links first.

Once you flesh it out more and polish it up, we can put it up, but we maintainers of the list need to have high standards on what is put here.

**Comment by besoeasy** [comment]
Reasonable Explanation, thanks for taking your time

**Pull request closed without merging**
